src/hooks/useSetting.ts

```diff
@@ -0,0 +1,12 @@
+import {
+  ThemeSettingsContext,
+  ThemeSettingsContextProps,
+} from "../providers/ThemeSettingsContext"
+import { useContext } from "react"
+
+// ----------------------------------------------------------------------
+
+const useSettings: () => ThemeSettingsContextProps = () =>
```

Comment:
change file and function name to `useThemeSettings`

Reply:
Also to Art's point, you could move this into the Theme provider file if you want to.

Reply:
updated